src/pages/gallery/index.tsx

```
@@ -0,0 +1,96 @@
+    setTags(tags);
+    setPalette(palette);
+  };
+
+  return { onUseTemplate };
+};
+
+const useTemplates = () => {
+  const [templates, setTemplates] = useState<Template[]>([]);
+
+  async function getTemplates(): Promise<Template[]> {
+    let { data: templates, error } = await supabase
```

Biome

[error] This let declares some variables that are only assigned once.

'templates' is never reassigned.

'error' is never reassigned.

Safe fix: Use const instead.


(lint/style/useConst)